### PR TITLE
feat(components): <Text> change font size via style props

### PR DESCRIPTION
**Pull request by sebald**

### CODE REVIEW

**Comment by sebald** [comment]
@ti10le we didn't talk about this but doing #1726 this would have been a nice featur :)

File: packages/components/src/Text/Text.tsx [hunk 1]
```diff
@@ -50,7 +52,14 @@ export const Text: PolymorphicComponentWithRef<TextOwnProps, 'span'> =
         {...props}
         as={as}
         variant={`text.${variant}`}
-        css={{ textAlign: align, color, cursor, outline, userSelect }}
+        css={{
```
Comment (ti10le) [inline packages/components/src/Text/Text.tsx:55]: @sebald should these props be able to be overwritten by the variant or not? 🤔

**Review by ti10le — CHANGES_REQUESTED** [review]

**Comment by ti10le** [comment]
Please add size prop to docs table :) Maybe to storybook, too.

**Review by ti10le — APPROVED** [review]

File: packages/components/src/Text/Text.tsx (hunk 1 above)
Comment (sebald) [reply packages/components/src/Text/Text.tsx:54]: I think not ... I hope not :D (since its `undefined`)

**Pull request merged**